mmv1/products/metastore/Service.yaml

```diff
@@ -236,6 +241,30 @@ properties:
         description: |
           Scaling factor, in increments of 0.1 for values less than 1.0, and increments of 1.0 for values greater than 1.0.
         required: false
+  - !ruby/object:Api::Type::NestedObject
+    name: 'scheduledBackup'
+    description: |
+      The configuration of scheduled backup for the metastore service.
+    properties:
+      - !ruby/object:Api::Type::Boolean
+        name: 'enabled'
+        description: |
+          Defines whether the scheduled backup is enabled. The default value is false.
+        default_value: false
```

Comment:
Is `default_value` required? Without `default_value`, will `enabled` has the value false in the API response?

Reply:
same as above, fails if not passed explictly

Reply:
In that case, `default_from_api` can be used here instead of `default_value`

Reply:
done